src/update.ts

```diff
@@ -77,38 +121,63 @@ export const update = async (shouldCommit = false) => {
         metadata[i.split(/:(.+)/)[0].trim()] = i.split(/:(.+)/)[1].trim();
       });
     }
-    if (metadata.start && metadata.end) {
-      let expectedDown: string[] = [];
-      let expectedDegraded: string[] = [];
-      if (metadata.expectedDown)
-        expectedDown = metadata.expectedDown
-          .split(",")
-          .map((i) => i.trim())
-          .filter((i) => i.length);
-      if (metadata.expectedDown)
-        expectedDegraded = metadata.expectedDown
-          .split(",")
-          .map((i) => i.trim())
-          .filter((i) => i.length);
```

Comment:
Additionally this PR I fixed the `expectedDegraded` array. You can see here we were actually just checking `expectedDown` twice, and assigning it to `expectedDegraded`